src/validation.h

```diff
@@ -505,6 +505,9 @@ class BlockManager {
         CBlockIndex** ppindex) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 };
 
+/** Global variable that points to the active CCoinsView (protected by cs_main) */
+extern std::unique_ptr<CCoinsViewCache> pcoinsTip;
```

Comment:
Did you remove it in last commit ? I still see it at d0e1e37

Reply:
Also comment in ATMP refers to it

Reply:
Good catch! Forgot to remove this after I split out the `CoinsTip()` rename commit.